packages/flat-lists/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "@react-native/flat-lists",
+    "version": "1.0.0",
```

Comment:
Same as the other package. Versioning here should be updated to `0.72.0`

Reply:
done